### ISSUE TITLE
Support terminal-table version 1.5.1

### ISSUE THREAD

**Issue by lion-man44** [presidentbeef/brakeman#709]
When I run `bundle install` to update `brakeman` and `i18n-tasks` , i get

```rb
Bundler could not find compatible versions for gem `terminal-table`:
  In Gemfile:
    i18n-tasks (= 0.8.7) ruby depends on
      terminal-table (>= 1.5.1) ruby

    brakeman (= 3.1.0) ruby depends on
      terminal-table (~> 1.4.5) ruby
```

I'm want to `i18n-tasks` and `brakeman` .
Could you update `terminal-table` , please?

**Comment by presidentbeef** [comment]
terminal-table 1.5.0 broke things, which is why the dependency is specified like it is.

In the meantime, you can try using the `brakeman-min` gem instead which declares minimal dependencies.

**Comment by lion-man44** [comment]
@presidentbeef Thank you for your advice.
Try out.

**Comment by aried3r** [comment]
> terminal-table 1.5.0 broke things

Could you specify which? Is there an issue at their repo to track?

**Comment by presidentbeef** [comment]
@aried3r [this commit](https://github.com/tj/terminal-table/commit/cb87f9872f659988f06fca21193fac64df938e08) changed headings from being a Row to an array of Rows. Fix is in #712.
